addon/-private/system/references/record.ts

```diff
@@ -74,7 +74,13 @@ export default class RecordReference extends Reference {
      let userRef = store.getReference('user', 1);
 
      // provide data for reference
-     userRef.push({ data: { id: 1, username: "@user" }}).then(function(user) {
+     userRef.push({ 
+       data: { 
+         id: "1", 
+         type: "user",
+         username: "@user" 
```

Comment:
this should be `attributes: { username: "@user" }` e.g. the full structure should be in `json-api` format:

```js
userRef.push({
  data: {
    id: "1",
    type: "user",
    attributes: {
      username: "@user"
    }
  }
})
```

Reply:
@runspired Added the attributes object